docs/plugin_developer_guide.md

```diff
@@ -313,87 +315,87 @@ func (demo *DemoPlugin) Run(context plugin.PluginContext, args []string){
+To keep user experience consistent, developers of Bluemix CLI plug-in should apply specific wordings, formats and colors to the terminal output. Bluemix CLI SDK provides the utility to help plug-in developers easily format and colorize the message output. We strongly recommend developers to comply with the following specifications so that the plug-ins are consistent with each other in terms of user experience.
 
-We tried to make the formats and colors to be similar to CF CLI too, to make the experience to be similar to CF as much as possible. 
+We tried to make the formats and colors to be similar to CF CLI too, to make the experience to be similar to CF as much as possible.
 
 ### 2.1. Global Specification
 
 1. Do NOT use "Please" in any message.
-  Correct:
-  <pre class="bx-console-block">Use '<span class="yellow">bluemix login</span>' to login first.</pre>
-  Invalid:
-  <pre class="bx-console-block">Please use '<span class="yellow">bluemix login</span>' to login first.</pre>
+   Correct:
+   <pre class="bx-console-block">First log in by running '<span class="yellow">bluemix login</span>'.</pre>
+   Invalid:
+   <pre class="bx-console-block">Please use '<span class="yellow">bluemix login</span>' to login first.</pre>
+
 2. Capitalize the first letter for all sentences and short descriptions. For example:
   <pre class="bx-console-block">Change the instance count for an app or container group.</pre>
   <pre class="bx-console-block">-i   Number of instances</pre>
+
 3. Add "..." at the end of "in-progress" messages. For example:
   <pre class="bx-console-block">Scaling container group '<span class="cyan">xxx</span>'...</pre>
+
 4. Use "plug-in" instead of "plugin" in all places.
 
-### 2.2. Plugin and Command Name
+### 2.2. Plug-in and Command Name
 
-To name the plugin for a service
+**To name the plug-in for a service**:
 
-- Use full name of the service in all lower case, and hyphen to replace space, for example 'my-service'
-- Use abbreviation only when it’s commonly accepted, for example use 'vpn' for Virtual Private Network service
+- Use full name of the service in all lower case, and hyphen to replace space, for example 'my-service'.
+- Use abbreviation only when it’s commonly accepted, for example use 'vpn' for Virtual Private Network service.
 
-To name the commands:
+**To name the commands**:
 
 - Use lower case words and hyphen
 - Follow a “noun-verb” sequence
-- For command that list objects, use plural of the object name (or "object-list” if plural won’t work)
-- Use common verbs like add, create, bind, update, delete …
+- For commands that list objects, use the plural of the object name, such as `objects`. If a plural will not work, use the name and list, such as `object-list`.
+- Use common verbs such as add, create, bind, update, delete …
 
-For example, the following command names are correct:
+The following command names are formatted correctly, and provides a possible description for the command:
```

Comment:
I added some explanatory text to each of the examples to help give people an idea of when they might use or not use a certain command name.